### DISCUSSION TITLE
[SOLVED] Laragon Links not working

### DISCUSSION THREAD

**Discussion by Starfruit7** [leokhoa/laragon#801] [Q&A]
![Screenshot 2024-06-05 230235](https://github.com/leokhoa/laragon/assets/72670090/a1fad341-8738-4e74-8c65-cc1ca5354ac8) 

I moved to laragon. I have my documentroot and mysql ready. But the problem is, the regular localhost link works but anything else, like http://localhost/masterlist , doesn't work.

It leads to a page that says:
The requested URL was not found on this server.

How can I fix this? 

Thanks!

**Comment by Biostate** [comment]
How did you fix this problem? Can you add details?
